_posts/2017-04-28-hello-world.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Hello World! Announcing the Drupal Pattern Lab working group.
+author: Evan Lovely
+---
+
+This last week at [DrupalCon Baltimore][dcon] a group of people passionate about using [Pattern Lab](http://patternlab.io) in our Drupal projects met and shared successes, dreams, and challenges. Some approaches have been shared, but many were re-inventing the wheel and tackling the same problems from different angles. Notable of those was the [Four Kitchens](http://fourkitchens.com) crew (Evan Willhite, Brian Lewis, and Randy Oest) with their [Emulsify theme](http://emulsify.info), [Salem Ghoweri](https://twitter.com/salem_ghoweri) of Senior Front-end Architect at Pegasystems, and yours truly: [Evan Lovely](http://evanlovely.com) of [Phase2](http://phase2technology.com), creator of [Pattern Lab Starter](https://github.com/phase2/pattern-lab-starter).
```

Comment:
Since you're linking to people, Brian's twitter is https://twitter.com/ModsUnraveled, Randy is https://twitter.com/amazingrando, and Evan doesn't use Twitter, but his d.o is https://www.drupal.org/u/evanmwillhite